Notifications docs (#246)
huggingface/hub-docs: docs/hub/notifications.md

# Notifications

> Notifications allow you to know when new activities (pull requests or discussions) happen on users or organizations you are watching.

Reviewer: ```suggestion
Notifications allow you to know when new activities (pull requests or discussions) happen on models, datasets and spaces belonging to users or organizations you are watching.
```

- A new discussion/PR, or comment is created in one of the repositories of an organization or user you are watching.

![Notifications page](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/notifications-page.png)

You'll get new notifications by email and [directly on the website](https://huggingface.co/notifications), you can change this in your notifications settings.

## Watching users and organizations

By default, you'll be watching all the organizations you are a member of and will be notified of any new activity on those.

You can also choose to get notified on arbitrary users or organizations, to do so use the "Watch" button on their HF profiles. Note that you can also quickly watch/unwatch users and organizations directly from the [notifications settings](#notifications-settings).

_Unlike Github, you cannot watch a specific repository. You must watch users/organizations to get notified about any new activity on any of their repositories. The goal is to simplify this functionality for users as much as possible and to make sure you don't miss anything you might be interested in._

## Notifications settings

> In your [notifications settings](https://huggingface.co/settings/notifications) page, you can choose specific channels to get notified on depending on the type of activity, for example, receiving an email for direct mentions but only a web notification for new activity on watched users and organizations. By default, you'll get an email and a web notification for any new activity but feel free to adjust your settings depending on your needs.

Reviewer: ```suggestion
In your [notifications settings](https://huggingface.co/settings/notifications) page, you can choose specific channels to get notified on depending on the type of activity, for example, receiving an email for direct mentions but only a web notification for new activity on watched users and organizations. By default, you'll get an email and a web notification for any new activity but feel free to adjust your settings depending on your needs.

Clicking the unsubscribe link in an email will unsubscribe you for the type of activity, eg direct mentions.
```

# Notifications

> Notifications allow you to know when new activities (pull requests or discussions) happen on models, datasets and spaces belonging to users or organizations you are watching.

Reviewer 2: ```suggestion
Notifications allow you to know when new activities (Pull Requests or discussions) happen on repositories belonging to users or organizations you are watching.
```

Reviewer 3: (nit) i think it's fine to keep "models, datasets and Spaces" to be more ML specific

By default, you'll receive a notification if:

- Someone mentions you in a discussion/PR.

> - A new comment is created in a discussion/PR you participated in.

Reviewer 3: ```suggestion
- A new comment is posted in a discussion/PR you participated in.
```

> - A new discussion/PR, or comment is created in one of the repositories of an organization or user you are watching.

Reviewer 2: ```suggestion
- A new discussion/PR or comment is created in one of the repositories of an organization or user you are watching.
```

![Notifications page](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/notifications-page.png)

> You'll get new notifications by email and [directly on the website](https://huggingface.co/notifications), you can change this in your notifications settings.

Reviewer 2: Maybe add a link to the notification settings at the end

## Watching users and organizations

> By default, you'll be watching all the organizations you are a member of and will be notified of any new activity on those.

Reviewer 3: ```suggestion
By default, you'll be watching all the organizations you are a member of and will be notified of any new activity on those.

**This is different from services like GitHub where you watch individual repos, here you watch users or organizations (and as a consequence you're notified on activity on their respective repos).**
```

Reviewer 3: Oh ok no read too fast, you mention this below already

> You can also choose to get notified on arbitrary users or organizations, to do so use the "Watch" button on their HF profiles. Note that you can also quickly watch/unwatch users and organizations directly from the [notifications settings](#notifications-settings).

Reviewer 2: ```suggestion
You can also choose to get notified on arbitrary users or organizations. To do so, use the "Watch" button on their HF profiles. Note that you can also quickly watch/unwatch users and organizations directly from the [notifications settings](#notifications-settings).
```
